zproject/urls.py

```diff
@@ -326,6 +326,7 @@
         PATCH=update_message_backend,
         DELETE=delete_message_backend,
     ),
+    rest_path("messages/ids", GET=get_messages_for_ids),
```

Comment:
Does it make sense to just a new path for it?  I feel this can be used powerfully to do innovative stuff.